process/interceptors/throttler/numThrottler.go

```diff
@@ -0,0 +1,41 @@
+package throttler
+
+import (
+	"sync/atomic"
+
+	"github.com/ElrondNetwork/elrond-go/process"
+)
+
+// NumThrottler implements InterceptorThrottler and limits the message processing if current counter > max
+type NumThrottler struct {
```

Comment:
this could be global, and to sit in core package. I would call numGoRoutineThrottler

Reply:
Agree, changed